src/components/new-request/search.vue

```diff
@@ -590,7 +595,7 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     this.setDesignation('')
     // clear "Select a Business Type" field when "View all business types" or Society is selected
     if (!this.entity_type_cd || this.entity_type_cd === EntityType.INFO) {
-      this.$refs.selectBusinessTypeRef.reset()
+      this.$refs.selectBusinessTypeRef && this.$refs.selectBusinessTypeRef.reset()
```

Comment:
Added this to pre-empt errors when selectBusinessTypeRef isn't defined (ie, component isn't rendered at the moment due to v-if).